modules/tinymce/src/themes/silver/test/ts/browser/throbber/ThrobberPopupTest.ts

```diff
@@ -43,6 +52,54 @@ describe('browser.tinymce.themes.silver.throbber.ThrobberPopupTest', () => {
     });
   });
 
+  context('context toolbar', () => {
+    beforeEach(async () => {
+      const editor = hook.editor();
+      editor.setContent('<p class="ctx-menu-me">Hello World</p>');
+
+      await Waiter.pTryUntil('Waiting for context toolbar to open', () => {
+        UiFinder.findIn(SugarBody.body(), '.tox-pop');
+      });
+
+      editor.setProgressState(true);
+      await pWaitForThrobber();
+    });
+
+    it('closed by throbber opening', async () => {
+      await Waiter.pTryUntil('context toolbar is closed', () => {
+        UiFinder.notExists(SugarBody.body(), '.tox-pop');
+      });
+    });
+
+    it('re-opens when the throbber closes', async () => {
+      const editor = hook.editor();
+      editor.setProgressState(false);
+      await Waiter.pTryUntil('context toolbar is open again', () => {
+        UiFinder.exists(SugarBody.body(), '.tox-pop');
+      });
+    });
+
+    it('does not re-open when the throbber closes - if there is no focus on the editor', async () => {
+      // blur the editor
+      const input = SugarElement.fromTag('input');
+      Insert.append(SugarBody.body(), input);
+      Focus.focus(input);
+
+      // close the throbber
+      const editor = hook.editor();
+      editor.setProgressState(false);
+      await UiFinder.pWaitForState('throbber is closed', SugarBody.body(), '.tox-throbber', Fun.not(Visibility.isVisible));
```

Comment:
```suggestion
      await UiFinder.pWaitForHidden('throbber is closed', SugarBody.body(), '.tox-throbber');
```
No need to over complicate this, as we already have a helper for this.